### PR TITLE
Follow-up Add documentation for NodeSwap tech preview feature

### PR DESCRIPTION
**Pull request by mburke5678**
Follow up to: https://github.com/openshift/openshift-docs/pull/41755

Preview: [Enabling swap memory use on nodes](https://deploy-preview-42225--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-swap-memory_nodes-nodes-working)

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio You are listed as the QE resource for this feature. Can you PTAL at this PR? I want to make sure I didn't change any of the technical details when I edited @ehashman 's original PR?

File: modules/nodes-nodes-swap-memory.adoc [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
 
-In {product-title}, you can configure nodes to use swap memory. By default, the Kubelet will not start in the presence of swap memory without further configuration.
+You can enable swap memory use for {product-title} workloads on a per-node basis. 
 
 [WARNING]
 ====
 Enabling swap memory can negatively impact workload performance and out-of-resource handling. Do not enable swap memory on master nodes.
 ====
 
+To enable swap memory, create a KubeletConfig to set the `swapbehavior` parameter. You can set limited or unlimited swap memory:
```
Comment (ahardin-rh) [inline modules/nodes-nodes-swap-memory.adoc:16]: `KubeletConfig` file (or CRD?) to set

**Comment by ahardin-rh** [comment]
Just one comment. LGTM!

**Comment by mburke5678** [comment]
@ehashman Can you review these changes when you return from PTO? We can create a new PR to fix anything. Wanted to merge this PR for docs freeze on 2/25.

**Pull request merged**